homeassistant/components/fortios/device_tracker.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support to use FortiOS device like FortiGate as device tracker.
+
+This component is part of the device_tracker platform.
+"""
+import logging
+import voluptuous as vol
+
+from fortiosapi import FortiOSAPI
+import homeassistant.helpers.config_validation as cv
```

Comment:
Please add a blank line between 3rd party and homeassistant import groups.

Reply:
done